scripts/process-konflux-prs/check-merge-safety.sh

```
@@ -37,7 +55,37 @@ if [[ ${#COMPONENTS[@]} -eq 0 ]]; then
+    local component="$3"
+    local check_name="Red Hat Konflux / ${component}-on-push"
+
+    local shas
+    shas=$(gh api "repos/${gh_repo}/commits?sha=${branch}&per_page=20" \
+        --jq '.[].sha' 2>/dev/null) || { echo "unknown"; return; }
+
+    while IFS= read -r sha; do
+        local conclusion
+        conclusion=$(gh api "repos/${gh_repo}/commits/${sha}/check-runs" \
+            --jq --arg n "$check_name" \
+            '([.check_runs[] | select(.name == $n)] | .[0].conclusion) // ""' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)